doc/instantsend.md

```diff
@@ -39,20 +39,32 @@ This value can be overridden by passing the following argument to the Dash Core
 -instantsenddepth=<n>
 ```
 
-The key thing to understand is that this value indicates the number of "confirmations" a successful Transaction Lock represents. When Wallet RPC commands are performed (such as `listsinceblock`) this attribute is taken into account when returning information about the transaction. The value in `confirmations` field you see through RPC is showing the number of `"Blockchain Confirmations" + "InstantSend Depth"` (assuming the funds were sent via InstantSend).
+The key thing to understand is that this value indicates the number of "confirmations" a successful Transaction Lock represents. When Wallet RPC commands which support `minconf` and `addlockconf` parameters (such as `listreceivedbyaddress`) are performed and `addlockconf` is `true`, then `instantsenddepth` attribute is taken into account when returning information about the transaction. In this case the value in `confirmations` field you see through RPC is showing the number of `"Blockchain Confirmations" + "InstantSend Depth"` (assuming the funds were sent via InstantSend).
 
-There is also a field named `bcconfirmations`. The value in this field represents the total number of `"Blockchain Confirmations"` for a given transaction without taking into account whether it was InstantSend or not.
+There is also a field named `instantlock` (that is present in commands such as `listsinceblock`). The value in this field indicates whether a given transaction is locked via InstantSend.
 
 **Examples**
 * InstantSend transaction just occurred:
-    * confirmations: 5
-    * bcconfirmations: 0
+    `listreceivedbyaddress 0 true`
```

Comment:
This breaks the list. Should probably split examples in two - `listreceivedbyaddress 0 true` and `listsinceblock` and then describe all the steps for each of them independently i.e.
1. `listreceivedbyaddress 0 true`
* InstantSend transaction just occurred
    * confirmations: 5
* InstantSend transaction received one confirmation from blockchain:
...
2. `listsinceblock`
...

Maybe it's also a good idea to add `listreceivedbyaddress 0` as a 3rd example and a comparison to `listreceivedbyaddress 0 true`.

Reply:
fixed